motoman_gp110_support/launch/test_gp110.launch

```diff
@@ -0,0 +1,9 @@
+<launch>
+  <include file="$(find motoman_gp110_support)/launch/load_gp110.launch" />
+
+  <node name="joint_state_publisher_gui" pkg="joint_state_publisher_gui" type="joint_state_publisher_gui" />
+
+  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
+
+  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find industrial_robot_client)/config/robot_state_visualize.rviz" required="true" />
```

Comment:
please remove the empty lines.

Reply:
Do you mean only the two between the nodes, not the one after the include?

Reply:
The three nodes you mean?

Yes, that would work.